src/workerd/server/log-schema.capnp

```diff
@@ -28,7 +28,8 @@ struct LogEntry {
   # Context depth for nested operations (optional, only included if > 0)
 
   enum LogLevel {
```

Comment:
I wonder if we even need the new enum here and can't just go with the existing and largely identical `rpc::Trace::Log::Level` in `src/workerd/io/worker-interface.capnp`? We can conveniently refer to it with a `using LogLevel = rpc::Trace::Log::Level;` alias, as done in trace.h.

Reply:
Almost identical, but not quite, the log level in worker-interface is based on JS style errors rather than c++ style errors.

JS has "log" C++ doesn't.
C++ has "fatal" JS doesn't.